### DISCUSSION TITLE
Telecamere e DVR analogico in HA

### DISCUSSION THREAD

**Discussion by mefrasa** [indomus/forum#199] [HUB: Home Assistant]
Buongiorno a tutti,
volevo qualche dritta su come visualizzare in HA le videocamere del mio sistema di sorveglianza di una casa secondaria.
Sono 6 telecamere collegate ad un DVR, che attualmente controllo con una delle tante app disponibili.

Grazie a chi potrà darmi qualche indicazione.

Saverio

**Comment by sky7176** [comment]
Ciao,
ma il DVR ha una connessione LAN?
Altrimenti la vedo dura, se non utilizzando un video server ad un canale a cui colleghi l'uscita video del tuo DVR.

Giuseppe

**Reply by mefrasa** [reply]
Ciao si, il dvr è collegato in lan e pubblicato su internet con un ddns

**Reply by sky7176** [reply]
Ciao, allora verifica che ci sia la possibilità del protocollo OVIF oppure prova a vedere se lo puoi integrare con VLC.
Che marcha è il tuo DVR?
Giuseppe

**Reply by mefrasa** [reply]
Ciao, la marca del mio DVR è LCT. Sto cercando se esistono dei flussi RSTP ma non riesco a trovarli. 
Io dall'esterno utilizzo tranquillamente l'app superLive per visualizzare le telecamere.

Grazie
Saverio

**Reply by sky7176** [reply]
Ciao,
ho trovato un manuale di un dvr che sembra quello che hai tu. 
Il flusso rtsp esiste e la porta è la 554, ma non so se posso caricare il Link del manuale.

**Reply by mefrasa** [reply]
Grande, grazie mille... penso che tu possa inserire qui il link (essendo un manuale tecnico non dovrebbero esserci problemi).
In alternativa puoi mandarmi il paragrafo dove parla del flusso rtsp?

Grazie ancora

**Reply by sky7176** [reply]
Ciao Saverio,
ti posto il link da cui puoi scaricare il manuale [Manuale](https://www.google.it/url?sa=t&source=web&rct=j&url=https://secpoint.it/Documenti_prodotti/Manuali/Documentazione%2520ELMO/Manuali%2520tecnici%2520TVCC/TVCC/IP/MT_DVR-NVR%2520PRO_2018-03.pdf&ved=2ahUKEwjkzvzthJ3wAhXJ8qQKHZ1aAyYQFjAFegQIGBAC&usg=AOvVaw2Ll7ccHPmiNkilLsecYlrB)

Mi sembra il tuo modello, i riferimenti li trovi a pagina 22 e a pagina 75.
Se tutto è configurato in modo corretto ti basta provare con VLC passandogli come parametro una cosa del genere, devi capire come dirgli il canale:
rtsp://[User]:[Password]@[Indirizzo IP NVR]:554

Sul mio HIKVISION la stringa è rtsp://[User]:[Password]@[Indirizzo IP NVR]:554/Streaming/Channels/101/

dove 101 è il canale 1 in alta definizione
102 canale 2
e via dicendo

Invece questo è quello che ho inserito in HA
camera.embedded_net_dvr_profilename001 dove 001 è il canale 1

**Reply by MrCi89** [reply]
Scusate l'intromissione, volevo chiedere come avevate risolto il problema. Grazie

**Comment by MrCi89** [comment]
Buonasera, mi accodo perchè anche io ho questo DVR. Sapresti dirmi per favore se e come hai risolto?
Grazie
